drivers/spi/spi_nxp_lpspi/spi_nxp_lpspi_common.c

```diff
@@ -301,7 +301,7 @@ int lpspi_configure(const struct device *dev, const struct spi_config *spi_cfg)
 	/* TODO: S32K3 errata ERR050456 requiring module reset before every transfer,
 	 * investigate alternative workaround so we don't have this latency for S32.
```

Comment:
```suggestion
	/* TODO: Investigate alternative workaround for Erratum ERR050456 to avoid
	 * module reset before each transfer and reduce latency.
```